### ISSUE TITLE
How to skip first iteration time in process bar ?

### ISSUE THREAD

**Issue by jiarenyf** [Lightning-AI/pytorch-lightning#19357]
### Description & Motivation

Because the first iteration time is incorrect (model initialization and torch.compile when first interation)

**Comment by awaelchli** [comment]
I agree it would be nice if we could re-init the progress bar automatically after the first or so iterations. With the current progress bar implementation, this is doable but not trivial.
